tests/general/InvalidLinkBearTest.py

```diff
@@ -83,6 +83,7 @@ def test_run(self):
         "https://github.com/coala/coala-bears/issues/200"
         'http://httpbin.org/status/203'
         ('http://httpbin.org/status/200').install_command()
+        `https://coala.io/200`
```

Comment:
https://coala.io/200 returns a 404 for me.

```
>>> r = requests.get("https://coala.io/200")
>>> r
<Response [404]>
```

How do the tests pass for this if this link is in the list of valid links?